tests/tasks/setup/eks/awscli-cp-with-vpc.yaml

```diff
@@ -0,0 +1,51 @@
+---
+apiVersion: tekton.dev/v1beta1
+kind: Task
+metadata:
+  name: awscli-eks-cluster-create-with-vpc-stack
+  namespace: tekton-pipelines  
+spec:
+  description: |
+    Create an EKS cluster.
+    This Task can be used to create an EKS cluster for a given service role, region in an AWS account 
+  params:
+  - name: cluster-name
+    description: The name of the EKS cluster you want to spin.
+  - name: kubernetes-version
+    default: "1.23"
+    description: The EKS version to install.
+  - name: region
+    default: "us-west-2"
+    description: The region where the cluster is in.
+  - name: endpoint
+    default: ""
```

Comment:
Do we want to put the prod endpoint for us-west-2 here? I am not sure if CX will know what to put here without some research.

Reply:
if we don't put anything, by default it points to prod endpoint.